.github/workflows/android-e2e-test.yml

```diff
@@ -38,8 +38,14 @@ jobs:
       - name: Install root node dependencies
         run: yarn install && yarn submodules
       - name: Install node dependencies
+        id: install_deps
         working-directory: ${{ env.WORKING_DIRECTORY }}
+        continue-on-error: true
         run: yarn install
+      - if: steps.install_deps.outcome == 'failure'
+        name: Reinstall node dependencies
+        working-directory: ${{ env.WORKING_DIRECTORY }}
+        run: yarn
```

Comment:
I guess you added this step because something was failing. Can you describe that issue here?

Reply:
Previously, when CI was building node dependencies, it was struggling with file linking of react-navigation, after we've been doing `yarn submodules` on a project (it was mostly throwing file not found error). What's strange is that after the second run, the error was disappearing since yarn probably already refreshed the filesystem tree? I don't know how it works under the hood, I also believe this error should be gone now, but I left it in case of any emergency (and also for keeping iOS and Android runners similar, since on iOS this step is already there!).

Reply:
Oki doki